paasta_tools/chronos_serviceinit.py

```diff
@@ -136,7 +136,10 @@ def _format_schedule(job):
     else:
         schedule = job.get("schedule", PaastaColors.red("UNKNOWN"))
     epsilon = job.get("epsilon", PaastaColors.red("UNKNOWN"))
-    formatted_schedule = "%s Epsilon: %s" % (schedule, epsilon)
+    schedule_time_zone = job.get("scheduleTimeZone", "null")
```

Comment:
I meant to defensively cater for a case where `job.get("scheduleTimeZone")` returns `None` (which might not present itself, agreed). The code treats `"null"` and `None` in the same way, defaulting to `UTC`...